From 31acf6b22b5b4ba4bacdcbba5c5b97263f8a1d17 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?M=C3=A1ir=C3=ADn=20Duffy?= <duffy@redhat.com>
Date: Sun, 25 Feb 2024 22:17:41 -0500
Subject: [PATCH] Create a jokes/puns skill (#1)
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

* Create a jokes/puns skill

Created by 8-year old Margaret.

* fixes to formatting, passes yamllint now

- adding | after each answer - this means the strings following don't have to be escaped, which should make it easier for a broader contributor base.
- adding the --- at the top
- removing some extraneous spaces
- removing extraneous empty line at end

Signed-off-by: Máirín Duffy <duffy@redhat.com>

* removing extraneous ' quotes

since all the quotes are escaped with |, the ' around the strings was
extraneous and would have shown up in the output. dropped them :)

Signed-off-by: Máirín Duffy <duffy@redhat.com>

---------

Signed-off-by: Máirín Duffy <duffy@redhat.com>
---
 .../writing/freeform/jokes/puns/qna.yaml      | 36 +++++++++++++++++++
 1 file changed, 36 insertions(+)
 create mode 100644 compositional_skills/writing/freeform/jokes/puns/qna.yaml

diff --git a/compositional_skills/writing/freeform/jokes/puns/qna.yaml b/compositional_skills/writing/freeform/jokes/puns/qna.yaml
new file mode 100644
index 0000000..a1e8845
--- /dev/null
+++ b/compositional_skills/writing/freeform/jokes/puns/qna.yaml
@@ -0,0 +1,36 @@
+---
+- answer: |
+    Why do birds eat wood?
+
+    Because they're peckish!
+  question: Tell me a pun about birds.
+- answer: |
+    Why do French people always eat snails?
+
+    They don't like fast food.
+  question: Tell me a pun about fast food.
+- answer: |
+    Why did the car have a belly ache?
+
+    Because it had too much gas!
+  question: Tell me a pun about gas.
+- answer: |
+    What did the ocean say to the ocean?
+
+    Nothing. It just waved!
+  question: Tell me a pun about waves.
+- answer: |
+    Do you want to hear a roof joke?
+
+    Nevermind. The first one's on the house!
+  question: Tell me a pun about roofs.
+- answer: |
+    What do dentists call their x-rays?
+
+    Tooth pics!
+  question: Tell me a pun about x-rays.
+- answer: |
+    Why does Peter Pan always fly?
+
+    Because he never lands.
+  question: Tell me a pun about Neverland.
-- 
GitLab

